src/server/google-cloud.md

```diff
@@ -50,6 +50,15 @@ so we recommend that you **use Cloud Run** instead of App Engine for new
 server-side Dart code.
 If you _want_ to use App Engine, consider using the [`appengine` package][].
 
+## Functions Framework for Dart
+
+The Dart Functions Framework is a Google-sponsored open source 
```

Comment:
Do you perhaps want to link to the pub package(https://pub.dev/packages/functions_framework) here with `[Dart Functions Framework][]`?

No need, just thinking of information/links that may be helpful.

Reply:
good idea!